test/exe/Main.hs

```diff
@@ -977,6 +978,30 @@ pluginTests = testSessionWait "plugins" $ do
       )
     ]
 
+cppTests :: TestTree
+cppTests = testSessionWait "cpp" $ do
+  let content =
+        T.unlines
+          [ "{-# LANGUAGE CPP #-}",
+            "module Testing where",
+            "#ifdef FOO",
+            "foo = 42"
+          ]
+  _ <- openDoc' "Testing.hs" "haskell" content
+  expectDiagnostics
+    [ ( "Testing.hs",
+        [ ( DsError,
+            (2, -1),
```

Comment:
I get the same line numbers as CI but a different failure as mentioned below. My guess is that this is because we’re using different CPP implementations (I’m using whatever the default is on linux when invoked via stack, I guess you might be on Windows or MacOS). I would like to avoid this test being flaky locally so maybe we can just test against both `(2,-1)` and `(2,0)` (by catching `HUnitFailure` on the first try) and succeed if either of them succeeds?

Reply:
Yeah, I'm doing this on my mac at the moment. What about this?